docs/monitoring.md

```diff
@@ -273,8 +309,8 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Download the event logs for all attempts of the given application as a zip file</td>
   </tr>
   <tr>
-    <td><code>/applications/[app-id]/[attempt-id]/logs</code></td>
-    <td>Download the event logs for the specified attempt of the given application as a zip file</td>
+    <td><code>/applications/[app-id]/logs</code></td>
+    <td>Download the event logs for the application as a zip file</td>
```

Comment:
this change looks wrong -- there are two endpoints, one for all attempts for the applications, and another for just one attempt.  The docs for the all-attempt endpoint is above this one.  with this change, its just duplicated.